index.html

```diff
@@ -0,0 +1,139 @@
+          <h2  class="pd1">
+            Worldwide broadcast E3 2021
+          </h2>
+          <p class="pheadline">
+            Hardware manufacturers, software developers, 
+            and publishers from the video game industry would have 
+            presented new and upcoming products. 
+          </p>
+          <h2 class="date">June.9.2020 (TUE) ~ 11 (FRY)</h2>
+          <p class="location">@ Los Angeles Convention Center, Los Angeles California</p>
+        </div>
+        <section class="events">
+          <h2 class="mainevents">Main Events</h2>
+          <h2 class="eventsline1">____</h2>
+          <div class="eventcard eventcard1">
+            <img class="eventimg" src="./imgs/SONY-SQUARE1.png" alt="">
+            <h2 class="event-tittle">Sony conference</h2>
+            <p class="eventdescpt">Sony ran its major reveal of the PlayStation 5 console with all 
+              the new games and sooner releases</p>
+          </div>
+          <div class="eventcard eventcard2">
+            <img class="eventimg" src="./imgs/nintendologo1.png" alt="">
+            <h2 class="event-tittle">Nintendo conference</h2>
+            <p class="eventdescpt">Nintendo surprises with Breath of the Wild 2, Banjo-Kazooie and more
+              This year's Nintendo Direct was packed with reveals</p>
+          </div>
+          <div class="eventcard eventcard3">
+            <img class="eventimg" src="./imgs/Electronic_Arts-Logo.png" alt="">
+            <h2 class="event-tittle">EA conference</h2>
+            <p class="eventdescpt">No more backwards compatible games for Xbox One and PS4
+             EA has released all the original games for all current consoles</p>
+          </div>
+          <div class="eventcard eventcard4">
+            <img class="eventimg" src="./imgs/ubisoft-logo1.png" alt="">
+            <h2 class="event-tittle">Ubisoft conference</h2>
+            <p class="eventdescpt">Ubisoft enters subscription battleground with Uplay Plus
+              Uplay Plus is Ubisoft's new subscription service that costs $14.99/month</p>
+          </div>
+          <button class="eventsbutton">See All Events</button>
+        </section>
+        <section class="speakers">
+          <h2 class="speakerstittle">Featured Speakers</h2>
+          <h2 class="eventsline1">____</h2>
+          
+        </section>
+        <section class="indexpartner">
+          <div>
+          <h2 class="partnertittle">Partner</h2>
+          <h2 class="partnerline">____</h2>
+          </div>
+          <ul class="partnerimgs">
+          <li><img class="partnerimg" src="./imgs/white-nvidia-logo-1png.png" alt=""></li>
+          <li><img class="partnerimg" src="./imgs/SONY-SQUARE1.png" alt=""></li>
+          <li><img class="partnerimg" src="./imgs/Electronic_Arts-Logo.png" alt=""></li>
+          <li><img class="partnerimg" src="./imgs/ubisoft-logo1.png" alt=""></li>
+          <li><img class="partnerimg" src="./imgs/nintendologo1.png" alt=""></li>
```

Comment:
- [ ] A good practice for `<img>` tag is always add a value for `alt` tag, please add it. 🐻